common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -156,6 +156,13 @@ div.problem {
   .question-description {
     @include margin(($baseline*0.75), 0);
   }
+
+  form > label {
+    display: block;
+    margin-bottom: $baseline;
+    font: inherit;
+    color: inherit;
+  }
```

Comment:
This looks too generic just like before, we already removed this from here https://github.com/edx/edx-platform/pull/13040/files#diff-c7047cb9d48f2f7570fe4f80d519a9fcL156